package-lock.json

```diff
@@ -2909,6 +2917,10 @@
         "once": "^1.4.0"
       }
     },
+    "envinfo": {
+      "version": "7.3.1",
+      "resolved": "https://registry.npmjs.org/envinfo/-/envinfo-7.3.1.tgz",
+      "integrity": "sha512-GvXiDTqLYrORVSCuJCsWHPXF5BFvoWMQA9xX4YVjPT1jyS3aZEHUBwjzxU/6LTPF9ReHgVEbX7IEN5UvSXHw/A=="
```

Comment:
Something got mangled here, the object should close.

Reply:
Oops!